relay-event-normalization/src/validation.rs

```diff
@@ -237,6 +244,9 @@ pub struct EventValidationConfig {
     ///
     /// If the event's timestamp lies further into the future, the received timestamp is assumed.
     pub max_secs_in_future: Option<i64>,
+
+    /// Controls whether the event has been validated before, in which case disables validation.
+    pub is_renormalize: bool,
```

Comment:
I chose `is_renormalize` because it's a known name and concept we use in normalization and librelay. Is `is_validated` more clear? I can update the name and expand docs if it helps.

Reply:
Should be addressed in [bd72909](https://github.com/getsentry/relay/pull/3214/commits/bd729097df816356731b6c53541eea860106119d), let me know if it's still unclear or there's a better wording for it.